client/src/components/chat-block/chat-block.tsx

```diff
@@ -0,0 +1,122 @@
+import { observer } from 'mobx-react';
+import React, { useEffect, useRef, useState } from 'react';
```

Comment:
React may be ommited, 
https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html